src/main/java/cs2103/v15_1j/jimjim/ui/JJUI.java

```diff
@@ -45,6 +47,11 @@ public void showTaskView() {
 	public void refreshUI(){
 		mainViewController.updateData(getDataLists());
 	}
+	
+	public void refreshUI(UIFeedback feedback){
+		mainViewController.updateData(getDataLists());
+		feedback.execute(mainViewController);
```

Comment:
are there any reasons why you choose to make an overloaded method like this instead of just calling the old refreshUI() and feedback.execute()?